certified-connectors/Progressus Advanced Projects/apiDefinition.swagger.json

```diff
@@ -0,0 +1,173 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Progressus Advanced Projects",
+    "description": "Use Progessus Advanced Projects connector to make odata calls to Progresssus Advanced Projects data in Power Automate or other tools.",
+    "contact": {
+      "name": "Todd Michael",
+      "url": "https://www.progressusglobal.com/contact-us/",
+      "email": "support@plumblineconsulting.com"
+    }
+  },
+  "host": "api.businesscentral.dynamics.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/manage/environments/906e989f-ea57-49a9-94fd-afc8aedf1fea/": {},
```

Comment:
@ProgressusGlobal All the empty operations can be removed from the swagger as they would not add any value to your connector.